mailing_list/models.py

```diff
@@ -138,15 +137,26 @@ def get_or_create_or_delete_mailing_lists_for_canvas_course_id(self, canvas_cour
                 mailing_list = MailingList(**create_kwargs)
                 mailing_list.save()
 
+            # cs_class_type is used to determine if the section
+            # is an enrollment section or a non-enrollment section.
+            # if it's null for a section with a real sis section id, we
+            # should consider it an enrollment section.
+            cs_class_type = None
+            if s['sis_section_id'] and s['sis_section_id'].isdigit():
```

Comment:
there may be a better way to do this, the idea is we will get sis_section_id's that exist in the course instance table those will have a numeric course instance id as well as ones that are strings i.e. "ext:12345". If we got a numeric (or a string that is numeric (is digit = True) we want to see if it exists in the course instance table. If it does, does it have a value for cs_class_type.